README.md

```diff
@@ -1 +1,72 @@
-# azure-cosmosdb-mongodbshardedcluster
\ No newline at end of file
+# Install MongoDB Sharding Cluster
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fcjsingh8512%2Fazure-cosmosdb-mongodbshardedcluster%2Fusers%2Fchsi%2Fazuredeploy.json" target="_blank">
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.png"/>
+</a>
+<a href="
+http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fcjsingh8512%2Fazure-cosmosdb-mongodbshardedcluster%2Fusers%2Fchsi%2Fazuredeploy.json" target="_blank">
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.png"/>
+</a>
+
+
+This template deploys a MongoDB Sharding Cluster on CentOS. It deploys 1 router server, one config server replica set with 3 nodes, and 1 shard which is a replica set with 4 nodes. So it totally deploys 8 nodes.
+
+The router server is exposed on public IP address that you can access through SSH on the standard port, also mongodb port 27017 open. You can access it for MongoDB data write and read.
```

Comment:
Do we need to enable SSH?  Mongo does not use SSH for any of its communications

Reply:
This is just enabled to do any debugging on the node if one wants to. Do you want me to disable it?

Reply:
Good idea.  Lets keep it